### ISSUE TITLE
1.16.3/4 Cause optifine to see through objects

### ISSUE THREAD

**Issue by ink717** [MrCrayfish/MrCrayfishVehicleMod#396]
![2021-01-22_17 03 49](https://user-images.githubusercontent.com/72815511/105561284-4c758080-5ce4-11eb-8b99-24698765c7eb.png)
After removing every single mods, its determined that Vehicle Mod cause the optifine shaders to see through almost everything. This happens on all shaders. The issue exists since early 1.16.3 all the way.

**Comment by Mahhew** [comment]
Yeah it's definitely an incompatibility with Optifine. I was successfully able to replicate this issue by damaging a vehicle and resolve it by repairing it or disabling Optifine. Though, if you really want to hold on to Optifine and Shaders, I'm pretty sure you can disable vehicle damage in the world/server config files.

**Comment by Erysch** [comment]
Try to disable Fast Render Option in OptiFine
